fastlane/Fastfile

```diff
@@ -250,7 +249,6 @@ platform :ios do
 
   desc "macOS build"
   lane :build_mac do |options|
-    check_pods
```

Comment:
and in turn these tests go faster and don't re-test the same thing independently